atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -54,7 +54,7 @@ public DbMetadataTable createMetadata(TableReference tableRef, ConnectionSupplie
     }
 
     @Override
-    public DbDdlTable createDdl(TableReference tableRef, ConnectionSupplier conns) {
+    public OracleDdlTable createDdl(TableReference tableRef, ConnectionSupplier conns) {
```

Comment:
ah, this is because we need `drop(CaseSensitivity)`. Is there a way to avoid having this explicitly, and/or will we revert this to be just a DbDdlTable when Postgres support is added? Mainly because it's not great for this to be inconsistent with everything else.

Reply:
I considered making it a part of DdlTable itself, but yeah Postgres impl doesn't really have that for now so it would just default to case sensitive, which would be misleading and I figured worse.

The other option was to just not use this table factory since the factory code is fairly simple :/